coin flipper untutorial v 1.0

truthgoddess opened the pull request:
Greetings @madduccino et al!

V 1.0 of what I initially called Coin Flipper is ready for your review. 

Let me know your thoughts so I can integrate them and get a better version ready for you and the curriculum. 

Cheers.

madduccino commented:
Looks great, @truthgoddess! I have some comments:

1. Add more comments to your code for teachers to reference and update the Woof version

2. I'm not sure this is necessary but I can be convinced! - "recommended that you create variables to represent things like the space between your buttons, the x values, and y values of each button. To make your buttons line up nicely, reference those variables and use math to define x and y values in terms of each other."

3. Do you think there would be value in gamifying this a bit more by creating a character that asks you to flip the coin or having something happen based on the result? Like, "do you think you'll get more heads than tails? You win." Just a thought.

4. Why are functions recommended in Step 4?

5. Should you explain the difference between experimental and theoretical probability?

6.  Slightly awkward sentence: "Make the first coin that appears appear on the upper left of the screen."

truthgoddess commented:
Hi @madduccino.

1. Done
2. Removed the text you mentioned.
3. Added a step at the end that encourages students to use the app to play a game with someone near them. Thought that might gamify it a bit. I feel it isn't a game, but a tool, but since apps are just as important as games, I think this might be a good way to keep the app focus, but the gamify note you gave?
4. Removed the rec. Yeah, I think if we recommend a programmatic method, we should explain why. Since I didn't, I removed it.
5. Explained at the start. 
6. Fixed.

Thanks. any tips and advice would be great. 

Also thought we could add a bonus step that is an autoflip button, but not sure it is necessary because I think it is hard enough.

madduccino commented:
Awesome work @truthgoddess !
